src/developer/web-api/tracker.md

```diff
@@ -187,9 +186,10 @@ of the relationship must conform to are based on the `Relationship Type` of the
 
 ### Attributes
 
-`Attributes` are the actual values describing the `tracked entities`. They can either be connected
-through a `tracked entity type` or a `program`. Implicitly this means `attributes` can be part of
-both a `tracked entity` and an `enrollment`.
+Attributes can be associated either through a tracked entity type or a program. This implies that
```

Comment:
I would still have an introduction sentence defining attributes.

`Attributes` are the values describing the `tracked entities`.`